patternfly/patternfly-elements
# fix: Accordion set to hidden when assigned to content set panel

## pfe-content-set

Bug: If an accordion was assigned directly to the pfe-content-set panel, it was always hidden.

Issue: [1239](https://github.com/patternfly/patternfly-elements/issues/1239)

### Preview


Link(s) to demo page(s) where this element can be viewed:
- [Broken example](https://happy-galileo-ea79c4.netlify.app/elements/pfe-content-set/demo/#test-custom-id) 
- [Fixed example](https://deploy-preview-1238--happy-galileo-ea79c4.netlify.app/elements/pfe-content-set/demo/#test-custom-id) 


### What has changed and why

   - Updated the template to add attributes to the tab and accordion used for rendering the content set
   - Updated the JS to target those new attributes along with the tag names


### Testing instructions

1. Go to https://deploy-preview-1238--happy-galileo-ea79c4.netlify.app/elements/pfe-content-set/demo/
2. Check that both the parent and child accordions are visible.


### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.


### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
